app/models/middleware_server.rb

```diff
@@ -42,11 +42,11 @@ def in_domain?
 
   # Returns whether a server is immutable or not.
   #
-  # By default, we define all servers as being immutable, so that power
-  # operations are not allowed, and it is the provider responsability to
+  # By default, we define all servers as being mutable, so that power
+  # operations are allowed, and it is the provider responsability to
   # reimplement this method if necessary.
   def immutable?
-    true
+    false
```

Comment:
Changing the hard-coded value from `true` to `false` will result in another issue. Running the power ops on dockerized servers. But agent doesn't allow that, so the MiQ ui will allow that but it fails. Why is the value hard-coded and not read from the properties? The idea is that all the hawkular servers running in the docker container have the immutable flag and they told it about themselves in the properties.

I don't get why there is always `true` or `false`, or is this method overriden somewhere?